lib/helpers.js

```diff
@@ -62,22 +61,22 @@ const format = (event, options = { ignoreSelection: false }) => {
     // transform entire file
     if (isEmbedded) {
       const re = /<script\b[^>]*>([\s\S]*?)(?=<\/script>)/gi;
-      editor.backwardsScanInBufferRange(re, bufferRange, iter => {
+      editor.backwardsScanInBufferRange(re, bufferRange, (iter) => {
         // Create new range with start row advanced by 1,
         // since we cannot use look-behind on variable-length starting
         // <script ...> tag
         const { start, end } = iter.range;
-        const startModified = [start.row + 1, start.column];
+        const startModified = [ start.row + 1, start.column ];
         const range = new iter.range.constructor(startModified, end);
-        transformRange(editor, range);
+        transformRange(editor, range, { ignoreErrors: options.ignoreErrors });
       });
     } else {
-      transformRange(editor, bufferRange);
+      transformRange(editor, bufferRange, { ignoreErrors: options.ignoreErrors });
```

Comment:
we should just pass `options` instead of redefining a new object every time like this